test/Tests/src/Data/Vector_Spec.enso

```diff
@@ -265,9 +267,9 @@ type_spec name alter = Test.group name <|
         boolvec.filter Filter_Condition.Is_False . should_equal [False]
 
     Test.specify "should filter elements with indices" <|
-        alter [0, 10, 2, 2] . filter_with_index (==) . should_equal [0, 2]
+        (alter [0, 10, 2, 2] . filter_with_index (==)) . should_equal [0, 2]
```

Comment:
I'm a bit confused why are these parentheses needed here? My understanding of the whitespace rules is that the previous expression looked ok.

Reply:
Right. No change on this line. I just decided to be consistent with the next two ones which needed additional parenthesis.